tests/CMakeLists.txt

```diff
@@ -243,7 +243,7 @@ endif()
 # AV2 tests
 
 if(AVIF_CODEC_AVM)
-    if(AVIF_LOCAL_GTEST)
+    if(AVIF_ENABLE_GTEST)
```

Comment:
Yannis: I think you meant to test `AVIF_ENABLE_GTEST` here. If it is correct to test `AVIF_LOCAL_GTEST` here, please add a comment because the reason is not obvious.